test/unit/deferred.js

```diff
@@ -861,15 +882,24 @@ QUnit.test( "jQuery.when(nonThenable) - like Promise.resolve", function( assert
 QUnit.test( "jQuery.when(thenable) - like Promise.resolve", function( assert ) {
 	"use strict";
 
-	var CASES = 16,
-		slice = [].slice,
+	var customToStringThen = {
+		then: function( onFulfilled ) {
+			onFulfilled();
+		}
+	};
+	if ( typeof Symbol === "function" ) {
```

Comment:
I decided to keep this one for its harmless property set, but the others are updated to skip tests unless `Symbol.toStringTag` is truthy.